css/main.css

```diff
@@ -32,6 +32,19 @@ nav .btn-login {
   float: right;
 }
 
+nav .button{
+  font-size: 20px;
```

Comment:
Doesn't the font size look too huge compared to rest of the page? Please try reducing the size and see which one looks better. After changing the font-size you'd also have to change the padding to compensate.
![image](https://user-images.githubusercontent.com/56217868/100311651-575ab000-2fd6-11eb-830d-29badb16de79.png)

Reply:
Okay let me do that